src/main/java/practice/StreamPractice.java

```diff
@@ -35,7 +48,11 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * Example: select men who can be recruited to army (from 18 to 27 years old inclusively).
      */
     public List<Person> selectMenByAge(List<Person> peopleList, int fromAge, int toAge) {
-        return Collections.emptyList();
+        Predicate<Person> personPredicate = person -> person.getAge() >= fromAge
+                && person.getAge() <= toAge && person.getSex() == Person.Sex.MAN;
```

Comment:
```suggestion
        Predicate<Person> personPredicate = p -> p.getAge() >= fromAge
                && p.getAge() <= toAge && p.getSex() == Person.Sex.MAN;
```
 Let’s  check it everywhere